docs/performance/sharding.md

```diff
@@ -57,48 +150,10 @@ When you have slightly more shards per table than CPUs, you ensure that query
 workloads can be parallelized and distributed maximally, which in turn ensures
 maximal query performance.
 
-However, if most nodes have more shards per table than they have CPUs, you
-could actually see performance degradation. Each shard comes with a cost in
-terms of open files, RAM, and CPU cycles. Smaller shards also means small shard
-indexes, which can adversely affect computed search term relevance.
-
-For performance reasons, one thousand shards per table per node is considered
-the highest recommended configuration. If you exceed this you will experience a
-failing cluster check.
-
-### Balancing allocation
-
-Finding the right balance when it comes to sharding will vary on a lot of
-things. And while it's generally advisable to slightly over-allocate, it's also
-a good idea to benchmark your particular setup so as to find the sweet spot.
-
-If you don't manually set the number of shards per table, CrateDB will make a best guess,
-based on the assumption that your nodes have two CPUs each.
-
-:::{TIP}
-For the purposes of calculating how many shards a table should be clustered
-into, you can typically ignore replica partitions as these are not usually
-queried across for reads.
```

Comment:
Great that we remove this as it is not valid. For balancing reads, replicas are used as well (randomly choosing primaries and replicas).

Reply:
I hadn't removed it, this statement was also relocated, but has been deleted now at your disposal per c28811f31. Maybe we should reverse the statement instead, in order to emphasize replica partitions MUST be taken into consideration when calculating shards?

Reply:
I see, thanks. Yes good point, but you mention this already at https://github.com/crate/cratedb-guide/pull/355/files#diff-957d0b41c07f17924adca70fdb58b1209c40bdd8b6958904d3e5707710f77a6eR99. Not sure if this should be highlighted even more, but won't hurt probably. I'll leave this up to you ;)

Reply:
I would actually mention and maybe point to the reference quide the use of replicas to not loose data when a node goes down, but also to speed up query throughput (concurrent reads on the same table) since replicas are used as well there.

Reply:
Thanks, let's follow up at https://github.com/crate/cratedb-guide/pull/355#pullrequestreview-3309268555, which includes your most recent suggestion from the other comment.